multisig-wallet/src/main.sw

```diff
@@ -124,7 +124,7 @@ impl MultiSignatureWallet for Contract {
         storage.nonce = storage.nonce + 1;
 
         match to {
-            Identity::Address(address) => transfer_to_output(value, asset_id, address), Identity::ContractId(contract) => force_transfer_to_contract(value, asset_id, contract), 
+            Identity::Address(address) => transfer_to_output(value, asset_id, address), Identity::ContractId(address) => force_transfer_to_contract(value, asset_id, address), 
```

Comment:
You can flatten this with the generic transfer function from the std, but that can be handled in another pr.

Reply:
Bumping versions should not be equated to functional upgrades or code quality. It's just a way to make the other projects work as expected.